code/modules/client/preferences/entries/character/names.dm

```diff
@@ -68,11 +78,18 @@
 			input += "[pick(GLOB.last_names)]"
 	return input
 
+/datum/preference/name/real_name/get_policy_link()
+	return CONFIG_GET(string/policy_naming_link)
+/datum/preference/name/real_name/get_policy_tooltip()
+	return CONFIG_GET(string/policy_naming_tooltip)
+
 /// The name for a backup human, when nonhumans are made into head of staff
 /datum/preference/name/backup_human
-	explanation = "Backup human name"
-	group = "backup_human"
 	db_key = "human_name"
+
+	name_type = "Alt. Human name"
+	tooltip = "In specific cases, this name is used instead of your character name."
```

Comment:
Ironically, isn't very specific and is kind of confusing. If you're going to use this kind of language, it'd be best to include at least one example. Otherwise being a tad more descriptive would be good.